src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1809,11 +1809,32 @@
       <Kind Name="SemicolonToken"/>
     </Field>
   </Node>
+
+  <Node Name="VariableDeconstructionDeclaratorSyntax" Base="CSharpSyntaxNode">
+    <Kind Name="VariableDeconstructionDeclarator"/>
+    <Field Name="OpenParenToken" Type="SyntaxToken">
+      <Kind Name="OpenParenToken"/>
+    </Field>
+    <Field Name="Variables" Type="SeparatedSyntaxList&lt;VariableDeclarationSyntax&gt;"/>
+    <Field Name="CloseParenToken" Type="SyntaxToken">
+      <Kind Name="CloseParenToken"/>
+    </Field>
+    <Field Name="EqualsToken" Type="SyntaxToken" Optional="true"> <!-- PROTOTYPE(tuples) should be Opt -->
+      <Kind Name="EqualsToken"/>
+    </Field>
+    <Field Name="Value" Type="ExpressionSyntax" Optional="true"/> <!-- PROTOTYPE(tuples) should be Opt -->
+  </Node>
+
   <Node Name="VariableDeclarationSyntax" Base="CSharpSyntaxNode">
-    <Kind Name="VariableDeclaration"/>
-    <Field Name="Type" Type="TypeSyntax"/>
+    <Kind Name="VariableDeclaration"/> <!-- If Kind is VariableDeclaration, then Type and Variables fields are set. -->
+    <Kind Name="DeconstructionDeclaration"/> <!-- If Kind is DeconstructionDeclaration, then DeconstructionVariables field is set and the Type may be var. -->
```

Comment:
The comment is out-of-date. I enumerated the various uses of `VariableDeclaration` in the PR description, including the case where `var` is stored in the `type` field.
That said, I'm probably remove the new `Kind`. Consumers will have to look at the fields to figure out which combination this is (or maybe I'll add a helper method).